src/content/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/setup-prometheus-agent.mdx

```diff
@@ -0,0 +1,235 @@
+---
+title: Set up the Prometheus agent
+tags:
+  - Integrations
+  - Prometheus integrations
+  - Install and configure Prometheus Configurator
+metaDescription: Set up the Prometheus agent
+---
+
+You need to place the examples below in the `config` section of the agent. Refer to the [installation method](/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/install-prometheus-agent) you used to known where that is in your case.
+
```

Comment:
first of all I would describe which is the default behavior.

We are saying something in the install page:
>By default, the New Relic Prometheus agent uses two annotations to discover targets: newrelic.io/scrape: "true" and prometheus.io/scrape: "true". All the targets in the cluster that are annotated with newrelic.io/scrape: "true" are discovered and scraped. However, the default annotation from the Prometheus community, prometheus.io/scrape: "true", is discovered but we filter them to only scrape the [Prometheus integrations](https://docswebsitedevelop-mangulonrdocswebsitekangmakesp.gtsb.io/docs/infrastructure/prometheus-integrations/integrations-list/integrations-list-intro) that we support.

but I would move it here instead

Reply:
Done